web/src/components/common/PdfViewer.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <v-sheet>
+    <iframe
+      v-if="supportsPDFs"
```

Comment:
Consider switching entire PDF render pipeline to using `pdfobject` instead of using `VuePdfEmbed`? See https://pdfobject.com/examples/#integrating-pdf-js